tests/libvmi/sidecar.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package libvmi
+
+import (
+	"fmt"
+
+	"kubevirt.io/kubevirt/tests/flags"
+)
+
+const (
+	HookSidecarImage = "example-hook-sidecar"
+)
+
+func RenderSidecar(version string) map[string]string {
```

Comment:
As Ed pointed out, could we make this as a `libvmi.Option`?
Also shouldn't this be something like `RenderSMBiosSidecar`?

Reply:
Please take a look here: https://github.com/kubevirt/kubevirt/pull/10574/files#diff-fe9ff27d97b7be1c5a5c7a3d04f614eda6c6c3afbeb8a9d38bdf90955917c69aR80